File: Daily Test/Python/Unique Sum - Integers With Alphabets 24.9.2025.py
n = int(input())
l = list(map(str, input().split()))
b = []
for i in l:
    j = 0
    p = ""
    c = ""
    while j < len(i):
        if i[j].isalpha():
            c += i[j]
        elif i[j].isdigit():
            p += i[j]
        j += 1
    b.append(c + p)
a = list(set(b))
g = [int(i[1:]) for i in a]
print(sum(g))

File: Daily Test/C++/Unique Sum - Integers With Alphabets 24.9.2025.cpp
#include <iostream>
#include <vector>
#include <string>
#include <set>
using namespace std;

int main() {
    int n;
    cin >> n;
    vector<string> l(n);
    for (int i = 0; i < n; ++i)
        cin >> l[i];
    set<string> a;
    for (int i = 0; i < n; ++i) {
        string c, p;
        for (char ch : l[i]) {
            if (isalpha(ch)) c += ch;
            else if (isdigit(ch)) p += ch;
        }
        a.insert(c + p);
    }
    int sum = 0;
    for (auto s : a) {
        sum += stoi(s.substr(1));
    }
    cout << sum << endl;
    return 0;
}